Source/debug.cpp

```diff
@@ -288,8 +263,7 @@ std::vector<DebugCmdItem> DebugCmdList = {
 	{ "help", "Prints help overview or help for a specific command.", "({command})", &DebugCmdHelp },
 	{ "give gold", "Fills the inventory with gold.", "", &DebugCmdGiveGoldCheat },
 	{ "give xp", "Levels the player up (min 1 level or {levels}).", "({levels})", &DebugCmdLevelUp },
-	{ "give spells", "Add all spells to player.", "", &DebugCmdGetAllSpells },
-	{ "give spells 10", "Set spell level to 10 for all spells.", "", &DebugCmdMaxSpellLevel },
+	{ "set spells", "Set spell level to {level} for all spells.", "{level}", &DebugCmdSetSpellsLevel },

```

Comment:
I would prefer that we follow the norm of `give spells`, `take spells`

Reply:
what's the point if you can do it with one just fine?

Reply:
A more generic interface to commands, making it easier to get an overview and discover options.
Some of this won't really take effect before we get a proper command prompt.